Sources/Amplitude/Plugins/iOS/UIKitUserInteractions.swift

```diff
@@ -26,17 +62,17 @@ class UIKitUserInteractions {
         addNotificationObservers
     }
 
-    @objc static func didBeginEditing(_ notification: NSNotification) {
+    @objc static func didEdit(_ notification: NSNotification) {
         guard let view = notification.object as? UIView else { return }
-        let userInteractionEvent = view.eventFromData(with: "didBeginEditing")
+        let userInteractionEvent = view.eventData.userInteractionEvent(for: "Edit")
         amplitudeInstances.allObjects.forEach {
             $0.track(event: userInteractionEvent)
         }
     }
 
-    @objc static func didEndEditing(_ notification: NSNotification) {
+    @objc static func didTouch(_ notification: NSNotification) {
         guard let view = notification.object as? UIView else { return }
-        let userInteractionEvent = view.eventFromData(with: "didEndEditing")
+        let userInteractionEvent = view.eventData.userInteractionEvent(for: "Touch")
```

Comment:
I would prefer to use event names that are a bit more specific - Something like blur or editing ended if trying for consistency cross platform, or something reminiscent of the platform names (didEndEditing, didSelectRow).

Reply:
Sounds good. Also, I think it's better to remove the row selection event entirely since there doesn't seem to be a straightforward way to get the selected row from the UITableView object in the notification.

Reply:
Check out the `hitTest:` apis if you have access to a uitouch.

Reply:
The notification does not seem to have any info about the UITouch, so I don't think we can use `hitTest:`.

Reply:
I'll mark this thread a resolved for now - we can discuss other approaches later since tracking row selection event does not seem to be that important at this point.